core/fsds/spacefs.go

```diff
@@ -52,7 +52,9 @@ func (d *SpaceFSDataSource) Get(ctx context.Context, path string) (*DirEntry, er
 
 	baseName := filepath.Base(path)
 	parentPath := filepath.Dir(strings.TrimRight(path, "/") + "/..")
-	parentEntries, err := d.service.ListDir(ctx, parentPath)
+
+	// TODO: Support non default bucket, passing "" for now which is default bucket
+	parentEntries, err := d.service.ListDir(ctx, parentPath, "")
```

Comment:
Yeah, with this ListBuckets approach, the FUSE fs logic would need some changes to account for the list of buckets are the root directory and then browse the files on subsequent subdirectories. I'll create a card to track this